tests/integration/tier1/system-up-to-date/test_system_up_to_date.py

```diff
@@ -0,0 +1,89 @@
+import os
+
+from envparse import env
+
+
+def test_versionlock_on_pkg(shell, convert2rhel):
+    """
+    System contains one package that is not updated and it's version
+    is locked. Display a warning about used version lock used.
+    """
+
+    # Try to downgrade two packages
+    assert shell("yum downgrade openldap wpa_supplicant -y").returncode == 0
+    assert shell("yum install -y yum-plugin-versionlock").returncode == 0
+
+    assert shell("yum versionlock openldap").returncode == 0
+
+    with convert2rhel(
+        ("--no-rpm-va --serverurl {} --username {} --password {} --pool {} --debug").format(
+            env.str("RHSM_SERVER_URL"),
+            env.str("RHSM_USERNAME"),
+            env.str("RHSM_PASSWORD"),
+            env.str("RHSM_POOL"),
+        )
+    ) as c2r:
+        # TODO do we want to test the scenario where we have a package locked by this?
```

Comment:
We could also run full conversion when the versionlock is applied. I would run together with some non updated package.